### ISSUE TITLE
Can not download files of course

### ISSUE THREAD

**Issue by codeforfun111** [nishad/udemy-dl-windows#159]
I have updated lastest version but still can not download all files of the course
Could you check it?
Thanks
![error](https://user-images.githubusercontent.com/30260533/28313568-8c8809b0-6be1-11e7-82ed-f25ff949f7a2.jpg)
Please prefix your issue with one of the following: [BUG], [PROPOSAL], [QUESTION]..

Replace everything between stars with current version of your udemy-dl.exe (output of udemy-dl.exe -v):  
Version : **X.X.X**

**In raising this issue, I confirm the following (please check boxes, eg [X]):**

- [ ] I have read and understood the [contributors guide](https://github.com/nishad/udemy-dl-windows/blob/master/.github/CONTRIBUTING.md).
- [ ] I have checked that the bug-fix I am reporting can be replicated, or that the feature I am suggesting isn't already present.
- [ ] I have checked that the issue I'm posting isn't already reported.
- [ ] I have checked that the issue I'm porting isn't already solved and no duplicates exist in [closed tab](https://github.com/nishad/udemy-dl-windows/issues?q=is%3Aissue+is%3Aclosed)
- [ ] I have checked the pull requests tab for existing solutions/implementations to my issue/suggestion.

**My familiarity with the project is as follows (check one, eg [X]):**

- [ ] I have never used the project.
- [ ] I have used the project briefly.
- [ ] I have used the project extensively, but have not contributed previously.
- [ ] I am an active contributor to the project.

**If the project maintainer has any additional requirements, they will be listed here.**

- No additional requirements.

---

{issue content here, replace this line with your issue content}

**Comment by vikasnewports** [comment]
Yes I am also facing this issue for attached files and sometime some videos are also giving error therefore I have to skip that video

**Comment by codeforfun111** [comment]
Not sure  the reason why?Some courses can downoad,but some courses can not download all files,

**Comment by carcelen** [comment]
![errror_udemy](https://user-images.githubusercontent.com/16994070/28493897-b486d108-6f1f-11e7-9342-9dd9e320e731.png)

**Comment by carcelen** [comment]
![error_udemy_download](https://user-images.githubusercontent.com/16994070/28493919-2585268e-6f20-11e7-924e-36affb90c383.png)

In my case, the error is similiar.... Somebody know where's the problem?? Thanks.

**Comment by sjabberwocky** [comment]
Probably this bug because of huge file path.
You could try -o d:\ to put you course on top of your disk.
It works for me.

**Comment by m0rpheux** [comment]
@sjabberwocky  i did that still get the error as @carcelen

**Comment by m0rpheux** [comment]
@carcelen  i just decided to download the pdf file because file path was too large and drop it on folder that it  should go  and then ran the commands again with the --lecture-start  and --lecture-end after the x file that got stuck on

**Comment by a2nmous** [comment]
I get same error @carcelen, anybody can solve the problem?

**Comment by JuanBarros2** [comment]
@sjabberwocky ty! it worked
